soh/soh/SohMenuBar.cpp

```diff
@@ -962,6 +962,8 @@ void DrawEnhancementsMenu() {
             UIWidgets::Tooltip("Always shows dungeon entrance icons on the minimap");
             UIWidgets::PaddedEnhancementCheckbox("Show Gauntlets in First Person", "gFPSGauntlets", true, false);
             UIWidgets::Tooltip("Renders Gauntlets when using the Bow and Hookshot like in OOT3D");
+            UIWidgets::PaddedEnhancementCheckbox("Colour Temple of Time's Medallions", "gToTMedallionsColours", true, false);
+            UIWidgets::Tooltip("Colours (or not) the Temple of Time Medallions\nThis needs a scene reload to take effect");
```

Comment:
So what you can do here, similar to our dirt path fix, is put a condition around the checkbox and call `PatchToTdMedallions();` from there, so you don't need a scene reload!

If that doesn't make sense, have a look at the `gSceneSpecificDirtPathFix` checkbox!

Reply:
Done! Thanks!